components/count-badge/count-badge.js

```diff
@@ -172,19 +293,37 @@ class CountBadge extends RtlMixin(LitElement) {
 		if (this.maxDigits && this.number.toString().length > this.maxDigits) {
 			numberString = `${'9'.repeat(this.maxDigits)}+`;
 		}
+		const iconSizeStyle = {};
+		if (this.icon && this._fixIconSizeTier1) {
```

Comment:
This size fixing is required for d2l-card-footer-link to cap all the icons at 18*18px. I don't expect any consumers to use this property directly. Any thoughts on keeping this private and not exposing the functionality in the readme?

Reply:
I think you can do this purely in CSS:

```css
:host([fixed-icon-size]) d2l-icon {
  --d2l-icon-height: 18px;
  --d2l-icon-width: 18px;
}
```